manticore/platforms/evm.py

```diff
@@ -880,6 +883,12 @@ def world(self):
     def check256int(value):
         assert True
 
+    @staticmethod
```

Comment:
Should not this be a core.util auxiliar method? Not really an EVM functionality.

Reply:
Fine with me.  I'll make the change.